words/predicates.txt

```diff
@@ -789,10 +894,12 @@ rapid
 rare
 raspy
 rattle
+raw
```

Comment:
I could see this being paired in a way that might be sexually suggestive to some people.